internal/zinc-core/src/main/scala/sbt/internal/inc/Incremental.scala

```diff
@@ -70,14 +70,15 @@ object Incremental {
      * Merge latest analysis as of pickling into pruned previous analysis, compute invalidations
      * and decide whether we need another cycle.
      */
-    def mergeAndInvalidate(partialAnalysis: Analysis, completingCycle: Boolean): CompileCycleResult
+    def mergeAndInvalidate(partialAnalysis: Analysis, registerCycle: Boolean): CompileCycleResult
```

Comment:
nit: consider renaming this to something like `shouldRegisterCycle` since `def registerCycle` exists, which this flag is encouraging the implementer to later call.